### PR TITLE
feat: ShareButtonConnected

### PR DESCRIPTION
**Pull request by vinnyjth**
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-ui-connected/src/ShareButtonConnected/ShareButtonConnected.js [hunk 1]
```diff
@@ -9,12 +9,21 @@ import { share } from '../utils';
 import ShareButton from './ShareButton';
 import GET_SHARE_CONTENT from './getShareContent';
 
-const ShareButtonConnected = ({ itemId, message, onPress, title, url }) => (
+const ShareButtonConnected = ({
```
Comment (redreceipt) [inline packages/apollos-ui-connected/src/ShareButtonConnected/ShareButtonConnected.js:12]:
```suggestion
// TODO: deprecate itemId prop
const ShareButtonConnected = ({
```

**Review by redreceipt — APPROVED** [review]
left a comment to add a deprecation note, merge it or don't either way, this looks good!

**Comment by redreceipt** [comment]
snaps dawg
